components/network/tor/Makefile

```diff
@@ -18,6 +18,7 @@ BUILD_BITS=		64
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		tor
+COMPONENT_REVISION=	1
 COMPONENT_VERSION=	0.4.1.9
```

Comment:
Just a nitpick: you put the COMPONENT_REVISION before COMPONENT_VERSION here

Reply:
Fixed :)